setup.py

```diff
@@ -45,16 +45,11 @@ def _read_long_description():
     version=GIT_VERSION,
     packages=find_packages(exclude=[f"{NAME}.tests", f"{NAME}.tests.*"]),
     entry_points={
-        "console_scripts": [
-            f"{NAME}_eval={NAME}.tests.evaluation.start:main",
-            f"{NAME}_capture_exitcode={NAME}.tests.evaluation.exitcode:main",
-            f"{NAME}_collect={NAME}.collect:main",
-        ],
         "pytest11": [
             f"{NAME_DASHED}={NAME}.plugin",
         ],
     },
-    install_requires=["pydriller", "coverage", "pytest"],
+    install_requires=["coverage", "pytest"],
```

Comment:
Shall we move `pytest-cov` from `DEV_REQUIRE` to `install_requires` or we use solely `coverage` API?

Reply:
Hmm. pytest-cov is not required in the code but I guess it's required to actually use this thing. I'll add it